src/sentry/templates/sentry/integrations/dialog-complete.html

```diff
@@ -14,7 +14,6 @@
   if (window.opener) {
     window.opener.postMessage({{ payload|to_json }}, {{ document_origin|safe }});
   }
-  window.close();
```

Comment:
do you have any idea why this was here in the first place?

Reply:
I have no clue why. I'm guessing as a convenience thing when the pipeline succeeds, but it was never intended to close for failures.